blackmouse572/quizler-fe
# Feat/join classroom

Test
1. Go to http://localhost:3000/classrooms/{id} to get code
2. Go to http://localhost:3000/classrooms
3. Click on Join classroom button 
4. Fill in the code

Expected:
- If error, should have toast to notifiy
- Else, join classroom, and classroom list should be refech and new classroom will display on screen
